grails-app/services/org/pih/warehouse/core/UserService.groovy

```diff
@@ -269,6 +269,9 @@ class UserService {
                         ilike("email", "%" + term + "%")
                     }
                 }
+                if (params.status) {
```

Comment:
Why it's called `status` instead of `active`?

Reply:
this same logic of calling this parameter _status_ was already used in a method `def findUsers(...)`   [on this line ](https://github.com/openboxes/openboxes/blob/f98db19ecfa175b8830f724d635d6154e4d3c9d1/grails-app/services/org/pih/warehouse/core/UserService.groovy#L293) so I just followed the convention that was already used in this situtation.